templates/podman-rootful.yaml

```diff
@@ -43,7 +43,7 @@ provision:
       mkdir -p /etc/tmpfiles.d
       echo "d /run/podman 0700 {{.User}} -" > /etc/tmpfiles.d/podman.conf
     fi
-    dnf -y install podman
+    dnf -y install --best podman && touch /etc/lima-podman
```

Comment:
`--best` is nice, but `touch /etc/lima-podman` seems controversial

Reply:
I don't see how using `/etc/lima-podman` can create any problems. Every service nowadays seems to store their config in there (including apps like `nanorc` or `screenrc`):

```console
$ lima ls /etc | wc -l
     189
```

But if you don't like it, how about `mkdir -p /var/lib/lima` then? I don't really care; I was only pointing out that a temp dir that a user may reasonably wipe clean is a poor location for a file that should **never** be deleted.